optaweb-vehicle-routing-frontend/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
Yes. I've decided to include them rather then ignore them:
- If we delete them, Cypress will generate them (not sure whether on each run or after installation/upgrade). Therefore we'd need to ignore them otherwise new users would end up with a dirty working try after cloning the repository and running Maven build.
- I didn't want to ignore example files like `cypress/plugins/index.js` because we would need to unignore them once we start using plugins (Employee Rostering does) and I don't like switching ignore patterns on and off (causes problems when switching between branches that have different ignore patterns).
- Sometimes it's nice to have these examples visible as they may help find solutions for configuration problems faster.